src/azul/service/manifest_controller.py

```diff
@@ -101,6 +104,7 @@ def get_manifest_async(self,
                            authentication: Optional[Authentication]):
 
         token = query_params.get('token')
+        auth_token = None if authentication is None else authentication.identity()
```

Comment:
The `azul.auth.Authentication` abstraction attempts to hide the specific authentication mechanism. We should reveal that as late as possible.